### ISSUE TITLE
Uploading an image in Base64 is not working anymore

### ISSUE THREAD

**Issue by blackhill** [xdan/jodit#494]
**Jodit Version:**  3.4.25

**Browser:**  Chrome
**OS:**  Windows

Uploading an image in Base64 is not working anymore.
This is also the case in the playground: https://xdsoft.net/jodit/play.html

Configuration used:
```js
"uploader": {
    "insertImageAsBase64URI": true
  }
```

**Expected behavior:**
Image should be uploaded and visible in the Editor

**Actual behavior:**
Nothing happened

**Comment by xdan** [comment]
https://xdsoft.net/jodit/play.html?uploader%5BinsertImageAsBase64URI%5D=true in this case it should only insert image as base64 (<img src="data:..."). Not upload.

**Comment by blackhill** [comment]
Yes it will not be uploaded as the image is inserted in base64 but the <img> element is not inserted in the Editor. (Sorry for the misunderstanding)

**Comment by mraulim** [comment]
@xdan I think the problem is here https://github.com/xdan/jodit/commit/6249491c9757365eef9fd1c7bf83977ab39db020#diff-15622f5301880e861dd96fb4d54a56f2 because the first argument of `defaultHandlerSuccess` should be Uploader: https://github.com/xdan/jodit/blob/05864812dd7aa1a93936895782147ff7d6a122db/src/modules/uploader/uploader.ts#L100

**Comment by satrimboli** [comment]
Running into the same issue. I was able to solve it by downgrading to `3.4.23`.

**Comment by voc0der** [comment]
https://github.com/xdan/jodit/issues/474 related?

**Comment by JamesPiggott** [comment]
I also have this problem using the latest version '3.4.29'. In fact, this problem also occurs on the website @ https://xdsoft.net/jodit/ and in the Playground. Currently, our clients manage to get around the problem by dragging the image into the editor.

From the comment above by @xdan I wonder if we are having disagreement between the terms 'upload' and 'insert'? Using the Upload button I expect the end result to be an image insertion into the text at the location of the cursor.

**Comment by dheerajawale-svg** [comment]
Is there any update on this issue. I am facing it in ["jodit-react": "^5.2.19",]

**Comment by xdan** [comment]
Closing during a backlog sweep — this dates back to jodit 2/3 and the codebase has moved multiple major versions since. If it's still relevant on 4.13.x, a fresh issue with current details will get looked at quickly.
